pkg/client/interfaces/types-analyze.go

```diff
@@ -8,7 +8,7 @@ package interfaces
 AnalyzeOptions contain all of the knobs and dials to control a read transcription
 from the Deepgram API
 
-Please see the documentation for live/streaming for more details:
+Please see the live/streaming documentation for more details:
```

Comment:
Should this say live/streaming? Probably should say Text Intelligence documentation

Reply:
ooops. good catch! copy and paste is not your friend in many cases (like this)